magicctapipe/resources/config.yaml

```diff
@@ -1,9 +1,11 @@
 mc_tel_ids:
     LST-1: 1
+    LST-2: 0
```

Comment:
in principle not writting at all LST-2 etc in the dictionary (like in the old card) should have the same effect because you loop over all the keys, but skip those that are not positive, right?

Reply:
This is correct only for LSTs. MAGIC-I and MAGIC-II entries are still mandatory, even if, for instance, we have MAGIC-I: 0.

Reply:
this got outdated because you introduced a function to check that all 4 LSTs are there, closing...